fs-sim/src/System/FS/Sim/STM.hs

```diff
@@ -31,9 +32,15 @@ runSimFS :: (MonadSTM m, MonadThrow m)
 runSimFS fs act = do
     var <- newTVarIO fs
     a   <- act (simHasFS var)
-    fs' <- atomically (readTVar var)
+    fs' <- readTVarIO var
```

Comment:
Are there no other occurrences of this pattern that we might want to change all at once?

Reply:
I found one other occurrence, so I've changed it there too